docs/kg.md

```diff
@@ -26,6 +26,7 @@ Usage:
   kg [command]
 
 Available Commands:
+  completion  generate the autocompletion script for the specified shell
```

Comment:
Ah nice this come for free with the newer version of cobra? That means kgctl also will have basic autocomplete :) it would be nice in the future to make this actually complete node and peer names